src/build/help/help.xml

```diff
@@ -2352,6 +2352,26 @@
 
                         <example>20150131-153358F_20150131-153401I</example>
                     </option>
+                    <!-- OPERATION - VERIFY COMMAND - OUTPUT OPTION -->
+                    <option id="output" name="Output">
+                        <summary>Output type.</summary>
+
+                        <text>
+                            <p>Output may be none (default) or text. Requesting text generates ouput to stdout. Output will always be written to the verify command logfile.</p>
+                        </text>
+
+                        <example>text</example>
+                    </option>
+                    <!-- OPERATION - VERIFY COMMAND - VERBOSE OPTION -->
+                    <option id="verbose" name="Verbose">
+                        <summary>Output verbose result.</summary>
+
+                        <text>
+                            <p>Verbose defaults to false and provides a minimal verify passed/verify failed response. True provides a more detailed response. </p>
```

Comment:
extra space before `</p>`?

Reply:
addressed in 1ab4e8eb20873848a78df60be92e2356098786ec